api/app/src/main/kotlin/packit/model/dto/SubmitRunInfo.kt

```diff
@@ -0,0 +1,8 @@
+package packit.model.dto
+
+data class SubmitRunInfo(
+    val name: String,
+    val branch: String,
+    val hash: String,
+    val parameters: Map<String, Any>?
```

Comment:
What are the possible types of parameters? Is it just string or number? Or even just strings (I think orderly parameters originally all got passed around as strings, but I might be misremembering!) But perhaps it's all more flexible now.

Reply:
it can be boolean, integer, number or string currently, orderly parameters were being passed around as string but i changed that because that doesnt make sense for when people want to input a number